src/components/Filter.js

```diff
@@ -27,23 +28,28 @@ DropdownToggle.propTypes = {
 
 export default class Example extends React.Component {
   render() {
+    const { onItemClick, onClick, dropdownOpen, selectedFilter } = this.props
     return (
       <ButtonDropdown
-        isOpen={this.props.dropdownOpen}
-        onClick={e => this.props.onClick()}
+        isOpen={dropdownOpen}
```

Comment:
Könnte man auch `isDropdownOpen` nennen. Finde das gut, Booleans immer mit `is...` oder `has...` oder `does...` beginnen zu lassen.

Reply:
Done!